lib/droplet_kit/client.rb

```diff
@@ -56,14 +57,19 @@ def resources
       @resources ||= {}
     end
 
+    def default_user_agent
+      "DropletKit/#{DropletKit::VERSION} Faraday/#{Faraday::VERSION}"
```

Comment:
Thoughts on adding Ruby or OS version?

Reply:
I decided against including the Ruby and OS version to not make this super convoluted, since ultimately what we care about is the library versions in question, I could be convinced otherwise